assets/src/components/TableAssignment.js

```diff
@@ -140,10 +151,33 @@ function CustomAssignmentTable (props) {
           if (isCurrentWeek) {
             borderAndCurrentWeekStyle['color'] = 'orange'
           }
+          // styling for assignment title column
+          if (j === 2) {
+            borderAndCurrentWeekStyle['width'] = '30%'
+            borderAndCurrentWeekStyle['maxWidth'] = 0
+            borderAndCurrentWeekStyle['overflow'] = 'hidden'
+            borderAndCurrentWeekStyle['textOverflow'] = 'ellipsis'
+            borderAndCurrentWeekStyle['whiteSpace'] = 'nowrap'
+            borderAndCurrentWeekStyle['borderRight'] = '0.1em solid rgba(224, 224, 224, 1)'
+          }
```

Comment:
Also adding the border between title and percent columns for visual identification of title association with percent column